driver.sh

```diff
@@ -64,6 +64,19 @@ setup_variables() {
       export ARCH=powerpc
       export CROSS_COMPILE=powerpc64le-linux-gnu- ;;
 
+    "ppc64")
+      config=powernv_defconfig
```

Comment:
Thanks for the patch!

The powernv defconfig is LE by default. We could build a BE powernv kernel, but instead we could get coverage of a different defconfig when booting BE. I suggest pseries.

We could build a pseries_defconfig, and boot it in a pseries qemu:
```
      config=pseries_defconfig
      qemu="qemu-system-ppc64"
      image_name=zImage.pseries
      qemu_ram=1G
      qemu_cmdline=( -machine s
                                  -nographic -vga none
                                  -initrd images/ppc64/rootfs.cpio )
```